ui/decredmaterial/tabs.go

```diff
@@ -1,71 +1,306 @@
 package decredmaterial
 
 import (
+	"image"
+	"image/color"
+
+	"golang.org/x/exp/shiny/materialdesign/icons"
+
+	"gioui.org/font"
+	"gioui.org/text"
+
+	"gioui.org/f32"
 	"gioui.org/layout"
+	"gioui.org/op/paint"
+	"gioui.org/unit"
 	"gioui.org/widget"
+	"gioui.org/widget/material"
+	"golang.org/x/image/draw"
 )
 
-// DefaultTabSize is the default flexed size of the tab section in a Tabs
-const DefaultTabSize = .15
+const (
+	Top Position = iota
+	Right
+	Bottom
+	Left
+)
 
+var adaptiveTabWidth int
+
+// Position determines what side of the page the tab would be laid out
+type Position int
+
+// TabItem displays a single child of a tab. Label and Icon in TabItem are optional.
```

Comment:
If both `Label` and `Icon` are optional, what does it display when neither is present.

Reply:
Just tab items with the minimum width of the active indicator is displayed.